pom.xml

```diff
@@ -81,14 +88,18 @@
       <groupId>com.jayway.jsonpath</groupId>
       <artifactId>json-path</artifactId>
       <version>2.1.0</version>
-    </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.plugins.workflow</groupId>
-      <artifactId>workflow-job</artifactId>

```

Comment:
Really should not have had this as a `compile`-scoped dep (i.e., runtime plugin dependency).